modules/modifying-template-for-new-projects.adoc

```diff
@@ -7,7 +7,7 @@
 [id="modifying-template-for-new-projects_{context}"]
 = Modifying the template for new projects
 
-As a cluster administrator, you can modify the default project template so that
+As a developer, you can modify the default project template so that
```

Comment:
I do not think this statement should be changed as this is about creating project template, not about network policies

Reply:
Can only cluster administrators create new projects?

Reply:
@JoeAldinger It is creating a project versus creating a project template. The project template once created by cluster admin can be used by developer.
https://docs.openshift.com/container-platform/4.16/applications/projects/configuring-project-creation.html